config/resources/domain.json

```diff
@@ -241,7 +241,7 @@
           "target": "codeList",
           "cardinality": "one"
         },
-        "template": {
+        "instruction": {
```

Comment:
I think 'template' was actually correct, according to this: https://data.test-vlaanderen.be/doc/applicatieprofiel/besluit-mobiliteit/#Variabele

So we should just update it in the frontend I think?